BCI tests failed-steps status changed from softfail to fail.

m-dati opened the pull request:
BCI test run 2 or more steps of tox runs that, when fail, report in the webgui `no fail` and only in a final collecting phase all failed steps as enumerated in a single one only failure box. So on the web report it is not easy to identify the previous failed step.

Scope of this PR is to change from _softfailed_ to **failed** the report box of a **tox failed step**, more evident and helpful in debugging analysis.

- Verification runs of code updated:
  - http://mdati-openqa.qe.suse.de/tests/665
  - clone of https://openqa.suse.de/tests/12570974

  - failed step VR
  - 
    - http://mdati-openqa.qe.suse.de/tests/665#step/bci_test_docker/33
    - http://mdati-openqa.qe.suse.de/tests/665#step/bci_test_docker/40
  - failed step cloned
    - https://openqa.suse.de/tests/12570974#step/bci_test_docker/33
    - https://openqa.suse.de/tests/12570974#step/bci_test_docker/40

  - passed step VR
    - http://mdati-openqa.qe.suse.de/tests/665#step/bci_test_podman/47
    - http://mdati-openqa.qe.suse.de/tests/665#step/bci_test_podman/54

  - passed step cloned
    - https://openqa.suse.de/tests/12570974#step/bci_test_docker/47
    - https://openqa.suse.de/tests/12570974#step/bci_test_docker/54

  - final fail VR
    - http://mdati-openqa.qe.suse.de/tests/665#step/bci_test_podman/62
  - final fail cloned
    - https://openqa.suse.de/tests/12570974#step/bci_test_docker/62

- other VRs:
  - http://mdati-openqa.qe.suse.de/tests/666
    - clone of http://openqa.suse.de/tests/12579333   
  - http://mdati-openqa.qe.suse.de/tests/668
    - clone of http://openqa.suse.de/tests/12579025

grisu48 commented on tests/containers/bci_test.pm:
```diff
@@ -36,18 +36,18 @@ sub run_tox_cmd {
     $cmd .= " --reruns $bci_reruns --reruns-delay $bci_reruns_delay";
     $cmd .= "| tee $tox_out";
-    record_info("tox", "Running command: $cmd");
+    record_info("tox $env", "Running command: $cmd");
```
This gets very soon very long and ugly. Let's please keep the old one in the title.

m-dati replied:
I suggest to keep at least the first word: I will add `(split(' ',$env))[0]`

m-dati replied:
VR
http://mdati-openqa.qe.suse.de/tests/677#step/bci_test_docker/56

m-dati replied:
VR
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18035

m-dati replied:
VR http://mdati-openqa.qe.suse.de/tests/678#

m-dati replied:
VR http://mdati-openqa.qe.suse.de/tests/678#step/bci_test_docker/56

grisu48 reviewed (commented):
One nitpick, otherwise LGTM

m-dati commented:
In last push, split added and code tidied

m-dati commented:
In last push, tidy executed and local unit [test](
https://github.com/m-dati/mytests/blob/split_test/perl/test.split.pl) of the split expression executed, see [output](https://github.com/m-dati/mytests/blob/split_test/perl/test.split.out), moreover the [VR](http://mdati-openqa.qe.suse.de/tests/678#step/bci_test_docker/56) PASS and readability is ok.

Being these cosmetic changes, but useful to BCI the test debugging, merge proceeding.

m-dati commented:
In last push, tidy executed, **first word** of strings split with any separator of  space/column/dash/underscore applied and local unit [test](
https://github.com/m-dati/mytests/blob/split_test/perl/test.split.pl) of the split expression executed, see [output](https://github.com/m-dati/mytests/blob/split_test/perl/test.split.out).
Finally the [VR](http://mdati-openqa.qe.suse.de/tests/678#step/bci_test_docker/56) PASS and readability is ok.

Being these cosmetic changes, but useful to BCI the test debugging, merge proceeding.

m-dati commented:
In last push, tidy executed, **first word** of strings split with any separator of  space/column/dash/underscore applied and local unit [test](
https://github.com/m-dati/mytests/blob/split_test/perl/test.split.pl) of the split expression executed, see [output](https://github.com/m-dati/mytests/blob/split_test/perl/test.split.out).
Finally the [VR](http://mdati-openqa.qe.suse.de/tests/678#step/bci_test_docker/56) PASS and readability is ok.

Being these cosmetic changes, but useful to BCI tests debugging, merge proceeding.
